packages/backup-format/src/version-2/index.ts

```diff
@@ -237,7 +237,7 @@ export function generateBackupVersion2(item: NormalizedBackup.Data): BackupJSONF
         const item: BackupJSONFileVersion2['posts'][0] = {
             identifier: id.toText(),
             foundAt: Number(data.foundAt),
-            postBy: data.postBy.toText(),
+            postBy: data.postBy?.toText() || 'person:localhost/$unknown',
```

Comment:
`postBy`is an optional type, do we really need this fallback? If so, can we use like `ProfileIdentifier.Unknow.toText()` instead?

Reply:
This field is now optional in the database, but not in the backup format. We should give it a reasonable fallback value, which is the old behavior.